data/json/monsters/zed-animal.json

```diff
@@ -383,6 +383,69 @@
     "melee_dice": 3,
     "dodge": 0
   },
+  {
+    "id": "mon_zow",
+    "type": "MONSTER",
+    "name": { "str": "zombie cow" },
+    "description": "Once a placid cow, this leathery horror stumbles and shudders, its pulsing black eyes scanning for prey beneath wicked horns.  It is big and heavy and murderous.",
+    "looks_like": "mon_cow",
+    "default_faction": "zombie",
+    "bodytype": "dog",
```

Comment:
I guess it should have cow bodytype?

Reply:
Not implemented yet to my knowledge, though I don't fully understand how bodytype is used

Reply:
Bodytype is used in some calculations, for grabs and snares when fighting monsters (and monsters fighting each other too). Like in this example:
![Screenshot from 2020-07-13 09-22-50](https://user-images.githubusercontent.com/7430004/87304505-4ad43000-c4eb-11ea-8694-40ade2499598.png)

Also it is used to determine the kind of armor friendly animals can wear, as it is today in Stable you can equip your cow (that is also "dog" bodytype) with a K-9 Kevlar harness.
![Screenshot from 2020-07-13 09-26-57](https://user-images.githubusercontent.com/7430004/87304730-a43c5f00-c4eb-11ea-903b-8411aa1f77a3.png)

Cow (or cattle) bodytype is not in the game as @LyleSY said, so for the purpose of having Zows i believe it's perfectly fine to replicate that value from the Original Cow. Looking forward to more zed fauna, i love playing wayward size 1 cities playthroughs.

Reply:
If I'm reading it right, maximum monster volume for this harness is 350 liters, while cows have 680 liters.